pkg/controller/node/cidr_set_test.go

```diff
@@ -163,46 +206,94 @@ func TestGetBitforCIDR(t *testing.T) {
 			expectErr:      false,
 		},
 		{
+			clusterCIDRStr: "be00::/8",
```

Comment:
Suggest you add a test name field to the structure, so that in the failure messages, the user can tell which test has actually failed.